### PR TITLE
Feature/remove sdx home in makefile

### PR DESCRIPTION
**Pull request by gemmai95**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,19 +1,12 @@
-dev: check-env
+dev:
 	cd .. && pip3 uninstall -y sdx-common && pip3 install -I ./sdx-common
 	pip3 install -r requirements.txt
 
 build:
-	pip3 install -I -r requirements.txt
-	git clone https://github.com/ONSdigital/sdx-common.git
```
Comment (JamesGardiner) [inline Makefile]: We still need to be getting sdx-common from somewhere. The agreed way in a dev environment is to git clone into the root of the directory then point pip at it, i.e:

```
$ git clone https://github.com/ONSdigital/sdx-common.git
$ pip install git+file://sdx-common.git@0.7.0
```

File: README.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ The following envioronment variables can be set:
 
 ### License
 
+
```
Comment (JamesGardiner) [inline README.md]: Is the new line needed?

File: Makefile (hunk 1 above)
Comment (necrophonic) [inline Makefile]: Raised a tech debt card to tidy up throughout repos

**Comment by necrophonic** [comment]
Superseded by cherry picked commits into #189

**Pull request closed without merging**